src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1434,16 +1438,24 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
         ///     This property forces all AST nodes that generated from this node
         ///     to be executed, even there is no change in AST nodes.
         /// </summary>
-        public virtual bool ForceReExecuteOfNode
+        [Flags]
+        public enum ExecutionHint
+        {
+            None = 0,
+            GenerateAst = 1,
+            ForceExecute = 3
+        }
```

Comment:
According to C# naming convention, `enum` that is used to serve as flags should be in plural form (more than one values can coexist), so it should be `ExecutionHints`.